lib/neo4j/core/cypher_session/adaptors/bolt.rb

```diff
@@ -12,6 +13,7 @@ class CypherSession
       module Adaptors
         class Bolt < Base
           include Adaptors::HasUri
+          include Adaptors::Schema
```

Comment:
Does this extraction still allow the gem to work for HTTP on versions of Neo4j before 3.0.0?  You couldn't query for everything through Cypher back then...

Reply:
This extraction is only for the purpose of enabling code sharing between bolt and java_driver adapter. The http adapter is not affected by this.